packages/react/src/alert-dialog/popup/AlertDialogPopup.tsx

```diff
@@ -90,10 +90,11 @@ const AlertDialogPopup = React.forwardRef(function AlertDialogPopup(
   const state: AlertDialogPopup.State = React.useMemo(
     () => ({
       open,
+      mounted,
```

Comment:
Not strictly related to these changes, but this now becomes public API - I feel like "mounted" doesn't describe the behavior very well, especially when `keepMounted = true`. "visible" could be a better option.

Reply:
Perhaps `hidden` to align with the DOM attribute that's being set?

Reply:
`visible` sounds good too though

Reply:
I think `hidden` makes the most sense to align with the attr. `visible` adds yet another term to deal with